### DISCUSSION TITLE
Allow sorting items in the create loan dialog

### DISCUSSION THREAD

**Discussion by grantfitzsimmons** [specify/specify7#5976] [Feature Requests]
<img width="828" alt="image" src="https://github.com/user-attachments/assets/1db78703-d03a-4c13-9ad3-522f4c2a5a66">

It would be useful to sort by Prep Type so the correct items can be chosen

**Requested by:** RBGE

**Comment by maxpatiiuk** [comment]
It isn't implemented in this dialog because those counts are fetched async.
This means that we either need to show a screen-wide loading bar until all counts are loaded, or to are going to see rows jumping around until all are loaded

as a compromise, we could make the headers clickable for sorting once all rows are loaded, but not store the sort preference in local storage so that it is back to unsorted next time it's opened
